fcrepo-auth-common/src/main/java/org/fcrepo/auth/common/ServletContainerAuthenticationProvider.java

```diff
@@ -177,12 +154,12 @@ public ExecutionContext authenticate(final Credentials credentials,
         } else {
 
             sessionAttributes
-                    .put(FedoraAuthorizationDelegate.FEDORA_USER_PRINCIPAL,
-                            EVERYONE);
+            .put(FedoraAuthorizationDelegate.FEDORA_USER_PRINCIPAL,
```

Comment:
If you are going to reformat this line, please go ahead and move the `.put(...)` up to line:156.